src/main/resources/marc4bibframe.yml

```diff
@@ -36,9 +32,21 @@ bibFieldRules:
           marc2ldCondition:
             controlFields:
               - tag: 008
-                data:
                 expression: args.contains(data.substring(22, 23))
                 args: a, b, c, d, e, f, g, j
+        - types: LANGUAGE_CATEGORY
+          parent: WORK
+          predicate: LANGUAGE
+          controlFields:
+            '008':
+              CODE: 35,38
+          marc2ldCondition:
+            controlFields:
+              - tag: 008
+                expression: data.substring(35, 38).trim().isEmpty() == false
+          constants:
+            LINK: http://id.loc.gov/vocabulary/languages/%CODE
+            TERM:
```

Comment:
Do we need this line? `TERM:`

Reply:
Looks like I didn't get this requirement:
![16-28-26-06-2024](https://github.com/FOLIO-EIS/lib-linked-data-marc4ld/assets/150008941/21e261e8-8d7d-4106-92be-9920161c9f0e)

Could you please clarify?

Reply:
Removed